tests/graphql/test_rosie.py

```diff
@@ -0,0 +1,76 @@
+"""
+Test for methods in graphql/rosie.py
+"""
+
+import unittest
+from unittest.mock import patch
+
+from codiga.graphql.rosie import graphql_get_rulesets, get_rulesets_name_string
+
+
+class TestRosie(unittest.TestCase):
+    """
+    Tests for graphql/rosie.py
+    """
+
+    def setUp(self):
+        pass
+
+    def tearDown(self):
+        pass
+
+    def test_graphql_get_rulesets_invalid_argument(self):
```

Comment:
Another test to do/make: check the return value of `graphql_get_rulesets`. For this, use the `return_value` of a mock. Mock the function `do_graphql_query` and then, assign a return value to the function by using `do_graphql_query_mock.return_value = <value>`.

See [this tutorial](https://realpython.com/python-mock-library/) and especially the part where they do `datetime.datetime.today.return_value = tuesday`.